tests/unit_tests/test_tethys_services/test_backends/test_arcgis_portal.py

```diff
@@ -0,0 +1,108 @@
+from unittest import mock
+from django import test
+from tethys_services.backends.arcgis_portal import ArcGISPortalOAuth2
+
+
+@test.override_settings(
+    OAUTH_CONFIG={
+        'SOCIAL_AUTH_ARCGIS_PORTAL_URL': 'https://my.test.arcgis/portal'
+    }
+)
+class ArcGISPortalBackendTest(test.SimpleTestCase):
+    strategy = mock.MagicMock()
```

Comment:
Somehow this comment got removed... anyway... move `strategy` into the `setUp()` method (as `self.strategy`) so it is reset between tests.

Reply:
Applied. Looks like test and coverage passed this time.